artpaint/windows/BrushStoreWindow.cpp

```diff
@@ -32,7 +32,7 @@
 #include <ScrollBar.h>
 #include <ScrollView.h>
 
-
+s
```

Comment:
The old sloppy "ALT+S" leaving an "s"... I do that all the time. :)

Reply:
ok, that annoys me, because that means I pushed without even building first - very sloppy and I didn't think I engaged in such behaviors.  Shameful.